.eslintrc.js

```diff
@@ -5,19 +5,23 @@ module.exports = {
     es6: true,
     jest: true,
   },
-  extends: ["plugin:react/recommended", "airbnb", "airbnb/hooks", "prettier"],
+  extends: [
+    "plugin:react/recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "airbnb",
+    "airbnb/hooks",
+    "prettier",
+    "plugin:neverthrow/recommended",
```

Comment:
will this throw an error if neverthrow is not used? if so, its going to cause too many errors. would not include

Reply:
agree, removed it [`9bf6aa8` (#3639)](https://github.com/dendronhq/dendron/pull/3639/commits/9bf6aa8febb3f8193b56bdd15a67d77f705f5ae4)